src/ImageSharp/Formats/Tiff/TiffDecoderOptionsParser.cs

```diff
@@ -122,15 +50,16 @@ public static void VerifyAndParseOptions(this TiffDecoderCore options, TiffFrame
                 }
             }
 
-            ParseCompression(options, entries.Compression);
-
             options.PlanarConfiguration = entries.PlanarConfiguration;
+            options.Predictor = entries.Predictor;
 
-            ParsePhotometric(options, entries);
-
-            ParseBitsPerSample(options, entries);
+            // todo: There is no default for PhotometricInterpretation, and it is required.
```

Comment:
i do not understand this TODO: What needs to be done here. I have seen this comment before your change, maybe this comment is no longer valid?

Reply:
PhotometricInterpretation doesn't have a default value, but I seem to have seen files (but I'm not sure) where this field may be missing. I wrote this comment for a long time, probably now we can delete it. At the moment case of missing this field wasn't confirmed in practice. I will delete it